src/discord_bot/bot.py

```diff
@@ -1,13 +1,21 @@
-import os
 import interactions
-import dotenv
+from pydantic_settings import BaseSettings, SettingsConfigDict
 from discord_bot.llm import answer_question
 
-dotenv.load_dotenv()
 
+class Settings(BaseSettings):
+  model_config = SettingsConfigDict(env_file="../../.env", env_file_encoding="utf-8")
+
+  DISCORD_BOT_TOKEN: str
+  AI_SERVER_URL: str = "http://localhost:8000"
```

Comment:
If things like `OPENAI_API_KEY` and `DISCORD_GUILD_ID` isn't needed here, please update it in the `.env.example`